cmd/osnadmin/main.go

```diff
@@ -54,7 +54,7 @@ func executeForArgs(args []string) (output string, exit int, err error) {
 	list := channel.Command("list", "List channel information for an Ordering Service Node (OSN). If the channelID flag is set, more detailed information will be provided for that channel.")
 	listChannelID := list.Flag("channelID", "Channel ID").Short('c').String()
 
-	remove := channel.Command("remove", "Remove an Ordering Service Node (OSN) from a channel.")
+	remove := channel.Command("remove", "Remove an Ordering Service Node (OSN)'s local joining block and ledger of a channel.")
```

Comment:
This isn't actually more clear to me... what specifically are you trying to solve?

The doc [here](https://hyperledger-fabric.readthedocs.io/en/latest/create_channel/create_channel_participation.html#add-or-remove-orderers-from-existing-channels) sounds correct:

> Add or remove orderers from existing channels
> 
> You can continue to use the osnadmin channel join and osnadmin channel remove commands to add and remove orderers on each channel according to your business needs. Be aware that before you remove a channel from an orderer, it is recommended that you first remove the orderer from the channel’s consenter set by submitting a channel update request.

The doc [here](https://hyperledger-fabric.readthedocs.io/en/latest/raft_configuration.html) sounds incorrect:

> To remove an ordering node from the consenter set of a channel, use the osnadmin channel remove command to remove its endpoint and certificates from the channel.

@tock-ibm What do you think? Does that latest doc paragraph need to be fixed?

Reply:
"Remove an OSN from channel" sounds to remove the OSN from the channel configuration.
Here the `osnadmin remove` is to remove the OSN's local ledger corresponding to a channel.

Reply:
Yes, that's exactly what i meant. The `osnadmin channel remove` cmd only removes the ledger data from an OSN locally, but the OSN is still a member of the channel, aka, it does not change the channel config.

And I can use the `"Remove a channel from an Ordering Service Node (OSN)."` as suggested.

Will update the pr. Thanks!